test/integration/scheduled-action-integration.ts

```diff
@@ -325,5 +392,67 @@ describe('Scheduled Actions API', async function () {
 
       expect(updatedAction.scheduledFor.timezone).to.equal('Europe/Berlin')
     })
+
+    describe('for AggregateRoot annotated entry', () => {
+      beforeEach(async () => {
+        const aggregateRootAnnotation: ContentTypeMetadata = {
+          annotations: {
+            ContentType: [
+              { sys: { id: 'Contentful:AggregateRoot', type: 'Link', linkType: 'Annotation' } },
+            ],
+          },
+        }
+        const environment = await testSpace.getEnvironment('master')
+        contentType = await environment.createContentType({
+          name: 'Page',
+          fields: [
+            {
+              id: 'title',
+              name: 'title',
+              type: 'Symbol',
+              localized: false,
+              required: false,
+            },
+          ],
+          metadata: aggregateRootAnnotation,
+        })
+        await contentType.publish()
+      })
+
+      afterEach(async () => {
+        await entry.delete()
+        await contentType.unpublish()
+        await contentType.delete()
```

Comment:
nit: should probably cancel the `scheduledActions` or we will see a spike in 404s (because the entry doesn't exist anymore)

Reply:
This is already happening in the cleanup method here: https://github.com/contentful/contentful-management.js/pull/1495/files#diff-905adda6bee733e47e8d05c1da61fab78e25a3b27e5bf38ccd1ef15eb5f6d8c2L13